fuzzers/005-tilegrid/generate_tiles.tcl

```diff
@@ -3,6 +3,7 @@ source "$::env(FUZDIR)/util.tcl"
 proc write_tiles_txt {} {
     # Get all tiles, ie not just the selected LUTs
     set tiles [get_tiles]
+    set not_allowed_sites [get_sites -of_objects [get_pblocks exclude_roi]]
```

Comment:
What is the purpose of the exclude_roi?  I haven't seen it before.

Reply:
The problem is that, for some parts (200T, for instance) there are some CLBs that cannot be used (they appear to be locked), and would generate errors in Vivado. With the exclude ROI, if set, we can exclude the problematic tiles from the tilegrid.

Reply:
But can the interconnect around them be used?  If so, we need to include the INT tiles in the tilegrid and tileconn.

Can you post the error from Vivado for reference?

If we keep the excluded ROI idea, there needs to be a large comment somewhere explaining why this happens.